### DISCUSSION TITLE
Thumbnail not generating for videos

### DISCUSSION THREAD

**Discussion by NTragus** [photoprism/photoprism#3762] [Q&A] [answered]
Hello,

I just got started with photoprism a few days ago, great app so far!

I've imported a few pictures and videos for test purposes, and I cannot get a thumbnail to show for the videos (black image only). When looking at the library logs, I get the message "index: found no preview image for ps1hh11nftojkgt3". There's one message per missing video, with a  unique identifier each time. 

I've tried following the checklist for broken thumbnails but to no avail, and docker logs did not yield any additional information appart from this "no preview image" message. I've tried re-generating thumbnails, but it only did so for pictures. 

Regarding the installation, I'm on a RPi 4B+ 4GB, using docker-compose. My files are stored on a Seafile server (self-hosted on the same RPi), that I reach thanks to Rclone plugin for Docker. 

I'd be grateful for any help regarding this topic!

Thank you

**Comment by lastzero** [comment, marked as answer]
Did you disable ffmpeg or generating jpegs in your configuration?

**Comment by NTragus** [comment]
I had the jpeg generation disabled. Changed the setting, re-indexed the library, all good now :)
I misunderstood this feature, I did not think it was related to thumbnails. 

Thanks for the quick answer!

**Reply by lastzero** [reply]
Maybe we should rename it to "Create Thumbnails" then or hide it completely.

**Reply by NTragus** [reply]
For most usage, I'd say there is no reason to disable thumbnails, the user would expect them to be there automatically, and see any missing thumbnail as a bug. 

My two cents would be to have this feature enabled by default, hide it, and keep it in the config file only as "advanced" stuff.

**Reply by lastzero** [reply]
We had a few purists in the beginning who didn't want us to create any files for whatever reason.
